### PR TITLE
sensitivity_models

### PR DESCRIPTION
**Pull request by katiebechman**
@AnnaB45 attached the cox model and define covariates files

### CODE REVIEW

File: analysis/000_define_covariates.do [hunk 1]
```diff
@@ -203,6 +203,22 @@ label values ethnicity ethnicity
...
+
+label define ethnicity 	1 "White"  					///
+						2 "South Asian"				///
+						3 "Black"  					///
+						4 "Mixed" 					///
+						5 "Other"					///
+						.u "Unknown"
+label values ethnicity ethnicity
+
+
```
Comment (AnnaB45) [inline analysis/000_define_covariates.do]: In the comments it says mixed is group 2 (line 208) but also mixed is group 6 (line 211), which one is right?

**Review by AnnaB45 — COMMENTED** [review]
@katiebechman - just one Q on the ethnicity groups

File: analysis/000_define_covariates.do (hunk 1 above)
Comment (katiebechman) [reply analysis/000_define_covariates.do]: Anna this is repetition. Line 207 to 219 need to be deleted. I will do this!

Comment (katiebechman) [reply analysis/000_define_covariates.do]: will ask James about the ethnicity code as he wrote this bit

Comment (katiebechman) [reply analysis/000_define_covariates.do]: spoken to James, code is correct. Mixed goes to 6, and then back to 4 [is moved directly to 4, then would have been recoded to black in line 210

**Comment by AnnaB45** [comment]
@katiebechman - I've tested locally and made a few more changes to the files. Can you have a look through and let me know if you're happy with these?

**Review by katiebechman — COMMENTED** [review]
@ Annab45

Yes these changes are brilliant, Thank you for generating the standinflix variables and changing the cox output for haemonc

**Pull request merged**